include/irr/builtin/materials/lambertian/singletexture/specializedshader_vertex

```diff
@@ -0,0 +1,18 @@
+#version 430 core
```

Comment:
instead of adding a `_vertex` suffix, do `.vert` suffix its a universally recognized extension